src/main/scala/essential_scala/Ex.scala

```diff
@@ -0,0 +1,126 @@
+package essential_scala
+
+object Ex extends App {
```

Comment:
If you'll use one app object for all the exercises, you can run into naming conflicts. Maybe you should use one object for one chapter?

Reply:
OK, I split file.